l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py

```diff
@@ -92,10 +92,10 @@ def check_periods(self):
     @api.multi
     def button_generate(self):
         self.date_generated = fields.Datetime.now(self)
-        auditfile_template = self._get_auditfile_template()
-        xml = auditfile_template.render(values={
-            'self': self,
-        })
+        xml = self.env.ref('l10n_nl_xaf_auditfile_export.auditfile_template')\
+            .render(values={
+                'self': self.sudo(),
```

Comment:
@Hayyan-Ebrahem as a final example, lets analyse this one:
1. Look at the [most recent version of this file on the 8.0 branch in the original OCA repo](https://github.com/OCA/l10n-netherlands/blob/8.0/l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py), and activate the [Blame view](https://github.com/OCA/l10n-netherlands/blame/8.0/l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py).
2. See that the newest OCA version is [not including this at the moment](https://github.com/OCA/l10n-netherlands/blame/8.0/l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py#L97)
3. In the [Blame view of that file in the Freshfilter community modules repo](https://github.com/sunflowerit/freshfilter-community-modules/blame/8.0/l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py), see that it was added by us in [this commit](https://github.com/sunflowerit/freshfilter-community-modules/commit/d8bba003bfb96ade711dbe92f8f7ed9787025fd8)
4. Decide to keep it